js/layout-javascript/agenda-code.js

```diff
@@ -1567,14 +1586,26 @@ DynamicList.prototype.getDateIndex = function (date) {
 };
 
 DynamicList.prototype.goToToday = function () {
-  this.goToDate(moment().format('YYYY-MM-DD'));
+  var dayUserLeft = parseInt(Fliplet.Navigate.query.dayUserLeft, 10);
+
+  if (dayUserLeft) {
+    this.goToDate(dayUserLeft, true);
```

Comment:
Why not just use:

```suggestion
    this.sliderGoTo(dayUserLeft);
```

Then there's no need to modify how `this.goToDate` works.